.github/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  - directory: /
+    open-pull-requests-limit: 5
+    package-ecosystem: gomod
+    schedule:
+      interval: weekly
+
+  - directory: /docs
+    open-pull-requests-limit: 5
+    package-ecosystem: npm
+    schedule:
+      interval: weekly
+
+  - directory: /
+    open-pull-requests-limit: 5
+    package-ecosystem: docker
+    schedule:
+      interval: weekly
+    
+  - directory: /
+    open-pull-requests-limit: 5
+    package-ecosystem: github-actions
```

Comment:
Nice, targeting the GH Actions as well 👌

Reply:
:ok_hand: cheers

Let's see - if we get a flood we can pull it back a bit!